src/components/maze/GameOverModalContent.tsx

```diff
@@ -58,14 +58,14 @@ export const GameOverModalContent = ({
 
     if (endGameResponse && !endGameResponse.ok) {
       toast({
-        title: 'Error Minting Cheddar',
+        title: 'Failed to add Cheddar to Farmed Balance.',
         status: 'error',
         duration: 9000,
         position: 'bottom-right',
         isClosable: true,
       });
     }
-  }, [endGameResponse, toast]);
+  }, [endGameResponse]);
```

Comment:
You must have the toast function in the array of dependencies so it doesn't generate conflicts in the building process

Reply:
okay, I will add it back